app/controllers/surveys_controller.rb

```diff
@@ -12,14 +12,13 @@ def create
     if @survey.save
       redirect_to new_survey_question_path(@survey)
     else
-      #redirect_to action: "new"
-      #redirect_to new_survey_question_path(@survey)
       render :new #// better way to show errors
     end
   end
 
   def show
    @survey = Survey.find(params[:id])
+   @first_question = @survey.questions.first
```

Comment:
Having a number in the varible and only looking for first in the db seems like a code smell. I would think more about the design